generator/spring-boot/src/test/java/io/fabric8/maven/generator/springboot/SpringBootGeneratorTest.java

```diff
@@ -71,6 +69,9 @@ public void javaOptions() throws IOException, MojoExecutionException {
         assertEquals(1, configs.size());
         Map<String, String> env = configs.get(0).getBuildConfiguration().getEnv();
         assertNull(env.get("JAVA_OPTIONS"));
+        } catch (Exception ex) {
+            ex.printStackTrace();
```

Comment:
can you make the test make error by rethrowing the exception?
Also if there is an exception the test will fail automatically and print the stack trace.

Reply:
Ah, that just for my debug and surely should be removed.
Another commit is coming soon...